src/wakefs/fuse.cpp

```diff
@@ -127,6 +129,32 @@ static bool collect_result_metadata(const std::string daemon_output, const struc
   return !result_ss.fail();
 }
 
+static void exec_in_pidns(pidns_args *nsargs) {
```

Comment:
I think this can be a const reference instead of a pointer.

Reply:
Also can you mark this function as `[[ noreturn ]]`

https://en.cppreference.com/w/cpp/language/attributes/noreturn

That will mainly make it clear that this function doesn't return. It also lets the compiler do some optimizations but that doesn't really matter here.

Reply:
> I think this can be a const reference instead of a pointer.

I need to pass `nsargs` into the clone function as a void pointer, is there a way to transform a reference into a pointer or is that not advisable?

Reply:
Sorry I didn't get back on this, A reference can have `&` called on it to make it a pointer which can then be cast to a void pointer.